src/NotificationsApp.vue

```diff
@@ -11,7 +11,7 @@
 				class="notifications-button__icon"
 				:size="20"
 				:title="t('notifications', 'Notifications')"
-				fill-color="var(--color-primary-text)" />
+				fill="currentColor" />
```

Comment:
Library icons don't have `fill` prop, only `fill-color`:

```js
    fillColor: {
      type: String,
      default: "currentColor"
    },
```

we could omit it here, as 'currentColor' already set as default

Reply:
right, copy paste error.
Also dropped the prop now that it is the default value.